docs/self_hosting/configuration/external_clickhouse.mdx

```diff
@@ -79,9 +81,22 @@ CLICKHOUSE_PORT=8123 # Change to your Clickhouse port if needed
 CLICKHOUSE_TLS=false # Change to true if you are using TLS to connect to Clickhouse. Otherwise, leave it as is
 CLICKHOUSE_PASSWORD=password # Change to your Clickhouse password if needed
 CLICKHOUSE_NATIVE_PORT=9000 # Change to your Clickhouse native port if needed
+CLICKHOUSE_CLUSTER=my_cluster_name # Optional: Set this if using an external Clickhouse cluster
 `
     ),
   ]}
 />
 
 Once configured, you should be able to reinstall your LangSmith instance. If everything is configured correctly, your LangSmith instance should now be using your external ClickHouse database.
+
+:::note
+Important considerations for clustered deployments:
+
+- Clustered setups must be configured on a fresh schema - existing standalone ClickHouse instances cannot be converted to clustered mode.
+- Clustering is only supported with externally managed ClickHouse deployments. It is not compatible with LangSmith-managed ClickHouse installations as these do not include required ZooKeeper configurations.
```

Comment:
`It is not compatible with self hosted Clickhouse installations installed through the LangSmith Helm, as these do not include required Keeper configurations.`